tests/integration/release/test_release.py

```diff
@@ -25,120 +25,102 @@
+    )
 
-    if expected_noise != "default":
-        config_dict[dataset_name][Keys.ROW_NOISE][NOISE_TYPES.omit_row.name][
-            Keys.ROW_PROBABILITY
-        ] = expected_noise
-        config = NoiseConfiguration(LayeredConfigTree(config_dict))
-    else:
-        config = NoiseConfiguration(LayeredConfigTree(config_dict))
-        # updating expected_noise from 'default' to actual default value
-        expected_noise = config.get_row_probability(dataset_name, NOISE_TYPES.omit_row.name)
-    assert isinstance(expected_noise, float)
+    # create unnoised dataset
+    dataset_name, dataset_func, source, year, state, engine = dataset_params
+    unnoised_data_kwargs = {
+        "source": source,
+        "config": NO_NOISE,
+        "year": year,
+        "engine": engine,
+    }
+    if dataset_func != generate_social_security:
+        unnoised_data_kwargs["state"] = state
+    unnoised_data = dataset_func(**unnoised_data_kwargs)
 
-    dataset_schema = DATASET_SCHEMAS.get_dataset_schema(dataset_name)
-    dataset = Dataset(dataset_schema, original_data, SEED)
-    NOISE_TYPES.omit_row(dataset, config)
-    noised_data = dataset.data
+    # In our standard noising process, i.e. when noising a shard of data, we
+    # 1) clean and reformat the data, 2) noise the data, and 3) do some post-processing.
+    # We're replicating steps 1 and 2 in this test and skipping 3.
+    full_dataset_name = DATASET_ARG_TO_FULL_NAME_MAPPER[dataset_name]
+    dataset_schema = DATASET_SCHEMAS.get_dataset_schema(full_dataset_name)
+    dataset = Dataset(dataset_schema, unnoised_data, SEED)
+    # don't unnecessarily keep in memory
+    del unnoised_data
+    dataset._clean_input_data()
+    # convert datetime columns to datetime types for _reformat_dates_for_noising
+    # because the post-processing that occured in generating the unnoised data
+    # in step 3 mentioned above converts these columns to object dtypes
+    for col in [COLUMNS.dob.name, COLUMNS.ssa_event_date.name]:
+        if col in dataset.data:
+            dataset.data[col] = pd.to_datetime(
+                dataset.data[col], format=dataset_schema.date_format
+            )
+            copy_col = "copy_" + col
+            if copy_col in dataset.data:
+                dataset.data[copy_col] = pd.to_datetime(
+                    dataset.data[copy_col], format=dataset_schema.date_format
+                )
+    dataset._reformat_dates_for_noising()
 
-    fuzzy_checker.fuzzy_assert_proportion(
-        name="test_omit_row",
-        observed_numerator=len(original_data) - len(noised_data),
-        observed_denominator=len(original_data),
-        target_proportion=expected_noise,
-    )
+    config = get_configuration()
 
-    assert set(noised_data.columns) == set(original_data.columns)
-    assert (noised_data.dtypes == original_data.dtypes).all()
+    for noise_type in NOISE_TYPES:
+        original_data = dataset.data.copy()
+        if isinstance(noise_type, RowNoiseType):
```

Comment:
Is this only testing row noise? If so I would rename the test accordingly.

Reply:
It is for now but it will later be updated to also test column noise and some post-processing tests.